packages/utils/src/file/convertImageToBlob/index.ts

```diff
@@ -0,0 +1,54 @@
+const TYPE_TO_FORMAT_MAPPER = {
+  png: 'image/png',
+  jpg: 'image/jpg',
+  jpeg: 'image/jpeg',
+  webp: 'image/webp',
+} as const;
```

Comment:
```suggestion
const TYPE_TO_FORMAT_MAPPER = {
  png: 'image/png',
  jpg: 'image/jpeg',
  jpeg: 'image/jpeg',
  webp: 'image/webp',
} as const;
```
- jpg는 호환되지 않아서 jpeg로 변경해줘야되지 않나요!?

Reply:
요게 단순히 넘겨서 실제로 실행하더라도 내부적으로 jpg는 jpeg로 처리한다고 이해했었는데 지원이 안된다면 수정해두겠습니다! 👍

Reply:
@Sangminnn 이 부분은 지원하지 않으면 `image/png`로 처리되더라구여! ㅎㅎ jpg랑 jpeg는 사실상 같은 포맷이다보니 그냥 jpeg로 처리하면 좋을 것 같다는 생각을했습니다!